deploy/deploy.ts

```diff
@@ -46,24 +148,26 @@ async function deployAll(
     console.log(`UpgradeableBeacon: ${beaconAddress}`);
     console.log(`Voucher implementation: ${await beacon.implementation()}`);
     console.log(`VoucherHub: ${voucherHubAddress}`);
-    return voucherHubAddress;
+    return { voucherHubAddress, voucherBeaconAddress: beaconAddress };
 }
 
 /**
  * Get deployment config according to chain.
  */
 async function getDeploymentConfig(chainId: number) {
     // Read default config of the target chain.
-    let pocoAddress: string = deploymentConfig[chainId]?.pocoAddress;
+    const config = deploymentConfig[chainId];
+    let pocoAddress: string = config?.pocoAddress;
     // Override config if required.
     if (process.env.IEXEC_POCO_ADDRESS) {
-        pocoAddress = process.env.IEXEC_POCO_ADDRESS;
+        config.pocoAddress = process.env.IEXEC_POCO_ADDRESS;
     }
     // Check final config.
     if (!ethers.isAddress(pocoAddress)) {
```

Comment:
```suggestion
    if (!ethers.isAddress(config.pocoAddress)) {
```

Reply:
Thanks! For all see [Verify poco address is set](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/35/commits/9e90c907c7cf72af3fc8806399521f6a2180a1cf)